sys/parsoid.js

```diff
@@ -182,6 +183,24 @@ PSP._wrapInAccessCheck = function(hyper, req, promise) {
     .then(function(responses) { return responses.content; });
 };
 
+function bodyToString(contentResponse) {
```

Comment:
This is equivalent to the .text() method in [the fetch spec](https://fetch.spec.whatwg.org/).